docs/CONTRIBUTING.md

```diff
@@ -1,25 +1,46 @@
 * If changing the C++ code, follow the [coding standard](https://endless-sky.github.io/styleguide/styleguide.xml).
+* Fill out the pull request template to the best of your ability. GitHub's pull request template features are not as thorough as issue templates. You're able to delete the entire pull request template and put in whatever you want; please do not do this. The purpose of the pull request template is to help facilitate the review, approval, and merging of pull requests.
 
 If proposing a major pull request, start by posting an issue and discussing the best way to implement it. Often the first strategy that occurs to you will not be the cleanest or most effective way to implement a new feature.
 
-## Closing issues
+### What to expect when opening a pull request
 
-If you believe your issue has been resolved, you can close the issue yourself. If your issue gets closed because a PR was merged, and you are not satisfied, please open a new issue.
+Endless Sky is developed by a community of volunteers. We're a collection of people with various different interests and skill sets, so the type of pull request you open will influence who will be looking at it.
+
+All pull requests go through a review process before being merged. We have a group of dedicated Reviewers who help provide reviews to pull requests as they come in. We also have a group of Developers who make determinations on what does and does not get merged, and are the ones who have the access to merge pull requests.
+* For smaller pull requests, such as typo fixes, it may only require a Developer seeing your pull request in order for it to be merged.
+* Larger pull requests typically receive reviews from multiple Reviewers and Developers before they are merged. That, in addition to the current backlog of open pull requests, can result in it taking months before a larger pull request is merged. We ask for your patience when waiting for your pull request to be reviewed.
+
+The type (and size) of pull request you open will also influence when it is eligable to be merged. We release updates alternating between "stable" and "unstable" versions.
+* During the development of an unstable release, any type of pull request is up for merging. Unstable releases are typically developed over the course of three to four months and with odd version numbers (e.g. v0.10.1).
+* During the development of a stable release, we typically only accept bug fixes and minor content tweaks. Stable releases are typically developed over the course of two to six weeks and with even version numbers (e.g. v0.10.2).
+
+If you've opened a new feature pull request during a stable release development cycle, it is unlikely to be merged until the stable release is finished. Or if you open a large pull request toward the end of an unstable release, it will likely have to wait until the next unstable release development cycle has begun before it has a chance to be merged.
```

Comment:
Suggested expansion because this timeline has on occasion been especially hard on some people who came into things blind.
```suggestion
If you've opened a new feature pull request during a stable release development cycle, it is unlikely to be merged until the stable release is finished. Or if you open a large pull request toward the end of an unstable release, it will likely have to wait until the next unstable release development cycle has begun before it has a chance to be merged.
Additionally, towards the end of an unstable release, Developers and Reviewers are often focused closely on a priority list determined in advance. A large pull request opened towards the end of an unstable release may not be reviewed at all until after the start of the next unstable release period. In this scenario your large PR may not receive much attention for upwards of 8 weeks after opening. Please understand that this may be unavoidable, and does not mean that the PR is valued less than anything else during that time. Please be especially patient if you find yourself in this unfortunate circumstance.
```

Reply:
Combining your and tibe's suggestions.
```suggestion
If you have opened a new feature pull request during a stable release development cycle, it is unlikely to be merged until the stable release is finished. Similarly, if you open a large pull request toward the end of an unstable release, it will likely have to wait until the next unstable release development cycle has begun before it has a chance to be merged.
Additionally, towards the end of an unstable release, Developers and Reviewers are often focused closely on a priority list determined in advance. A large pull request opened towards the end of an unstable release may not be reviewed at all until after the start of the next unstable release period. In this scenario your large PR may not receive much attention for upwards of eight weeks after opening. Please understand that this may be unavoidable, and does not mean that the PR is valued less than anything else during that time. Please be especially patient if you find yourself in this unfortunate circumstance.
```